project/AppDependencies.scala

```diff
@@ -9,12 +9,12 @@ object AppDependencies {
     "uk.gov.hmrc"                %% "bootstrap-frontend-play-28" % "5.20.0",
     "uk.gov.hmrc"                %% "auth-client"                % "5.8.0-play-28",
     "uk.gov.hmrc"                %% "play-frontend-hmrc"         % "0.94.0-play-28",
-    "uk.gov.hmrc.mongo"          %% "hmrc-mongo-play-28"         % "0.60.0",
+    "uk.gov.hmrc.mongo"          %% "hmrc-mongo-play-28"         % "0.62.0",
     "com.github.pureconfig"      %% "pureconfig"                 % "0.17.1",
     "com.beachape"               %% "enumeratum-play"            % "1.7.0",
     "org.webjars.npm"            %  "accessible-autocomplete"    % "2.0.3",
-    "uk.gov.hmrc"                %% "play-language"              % "5.1.0-play-28",
```

Comment:
Should this not remain on version "5.1.0-play-28" ?
See:
https://catalogue.tax.service.gov.uk/dependencies/merchandise-in-baggage-frontend/0.539.0

Reply:
Not sure, but have reverted the change